app/controllers/concerns/workflow_execution_actions.rb

```diff
@@ -262,4 +264,12 @@ def search_params
     search_params[:name_or_id_cont] = params.dig(:q, :name_or_id_cont)
     search_params
   end
+
+  def namespace_path
+    if @workflow_execution.namespace.type == 'Group'
```

Comment:
This can be changed to use `.group_namespace?` and the elsif can be changed to use `.project_namespace?`